index.js

```diff
@@ -67,6 +67,7 @@ function init (ssb, config) {
 
   /* secret keys store / helper */
   const keystore = {} // HACK we create an Object so we have a reference to merge into
+  // TODO: ssb-box2 stores the keyring at config.path/keyring. we're gonna have to merge this right?
```

Comment:
may need to migrate contents, or expose option in ssb-box2

Reply:
let's make a followup issue to do this in another PR

Reply:
made issue https://github.com/ssbc/ssb-tribes/issues/97